pkg/ebpf/tracer.go

```diff
@@ -21,7 +21,7 @@ import (
 
 var (
 	expvarEndpoints map[string]*expvar.Map
-	expvarTypes     = [4]string{"conntrack", "state", "tracer", "ebpf"}
+	expvarTypes     = [5]string{"conntrack", "state", "tracer", "ebpf", "kprobes"}
```

Comment:
This might be a _ton_ of expvar metrics. Can we just take a sum? Or filter out the ones we care about?

Reply:
can we do that on the expvar check instead ? (so those metrics are still available when manually hitting the endpoint)

Reply:
Yes, that sounds fine to me :)